src/control/PlanarArmController.h

```diff
@@ -41,7 +41,7 @@ class PlanarArmController {
 		// NOTE: currJointPos could extend beyond the safetyFactor, so safety factor
 		//       normalization logic is performed.
 		set_setpoint(currJointPos);
-		CHECK_F(safetyFactor > 0.0 && safetyFactor < 1.0);
+		assert(safetyFactor > 0.0 && safetyFactor < 1.0);
```

Comment:
Why the change from `CHECK_F` to `assert`? In general we should usually prefer `CHECK_F`, since loguru has some nicer features, and still runs the checks in release mode.